util/rlp/fuzz/std/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "rlp-std"
```

Comment:
just a quick implementation to confirm the panic works under a stable build. I will retract it since it will be a unit test.